sdk/spring/ci.cosmos.yml

```diff
@@ -0,0 +1,584 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/spring/
```

Comment:
Should we have this only run on sdk/spring/azure-spring-data-cosmos instead of all of spring @kushagraThapar ?